README.md

```diff
@@ -70,6 +72,40 @@ cd scripts/user_scripts
 ./train_model.sh
 ```
 
+More details for each of these steps are described more detail in the following sections.
+
+#### Data Preprocessing
+
+This repository uses [Dask](https://docs.dask.org/en/stable/) to load and process data. Our code is configured to load datasets into Dask from Parquet files. If using a different file format, this may need to be changed.
+
+##### Downloading Data
+
+This repository is designed to utilize datasets available through the HuggingFace Hub. There are two ways to download data:
+
+1. [download_data.py](./src/download_data.py), run via the [download_data.sh](./scripts/download_data.sh) script, can download data through the HuggingFace Filesystem. To do this, ensure that the correct parameters are set in the configuration YAML, specifically `hf_filesystem_path`. This must correspond to the correct HF Filesystem path. This method is good for smaller datasets that can easily fit in memory.
```

Comment:
Replace 

"download data through the HuggingFace Filesystem."

with

"download data through the [HuggingFace Filesystem].(https://huggingface.co/docs/huggingface_hub/v0.22.0.rc0/en/package_reference/hf_file_system#huggingface_hub.HfFileSystem)"